packages/components/src/Pagination/Pagination.module.scss

```diff
@@ -51,15 +51,3 @@
     }
   }
 }
-
-// Truncate indicator
-.truncateIndicatorWrapper {
```

Comment:
Wasn't being used at all, It got split out into it's own component and I guess they forgot to delete this